src/compiler/types.ts

```diff
@@ -6449,6 +6450,7 @@ namespace ts {
     export interface CommandLineOptionOfListType extends CommandLineOptionBase {
         type: "list";
         element: CommandLineOptionOfCustomType | CommandLineOptionOfStringType | CommandLineOptionOfNumberType | CommandLineOptionOfBooleanType | TsConfigOnlyOption;
+        listPreserveFalsyValues?: boolean;

```

Comment:
I dont think we should add this part..

Reply:
I added this because the list type doesn't support blank string entries, which are required for this feature.

Reply:
Don't understand. Why is `opt.listPreserveFalsyValues ? v : (v || "")` not same as `v || ""` for empty string?

Reply:
This case isn't needed. I reverted it. The other cases are, as they filter out empty strings.